charts/zenko/requirements.yaml

```diff
@@ -14,10 +14,12 @@ dependencies:
   repository: "http://storage.googleapis.com/kubernetes-charts-incubator"
   alias: zenko-queue
 - name: zookeeper
-  version: "1.0.0" 
+  version: "1.0.0"
   repository: "http://storage.googleapis.com/kubernetes-charts-incubator"
   alias: zenko-quorum
-
+- name: redis-ha
```

Comment:
You could add a condition to this chart so it could be disabled in the single node values.

Reply:
Same here.